<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="40753" started-at="2020-04-27T12:03:02Z" finished-at="2020-04-27T12:03:43Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="40753" started-at="2020-04-27T12:03:02Z" finished-at="2020-04-27T12:03:43Z">
      <class name="com.xm.qa.test.AddGroupTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.xm.qa.test.AddGroupTest@4313f5bc]" name="setup" is-config="true" duration-ms="14896" started-at="2020-04-27T17:33:02Z" finished-at="2020-04-27T17:33:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="addGroup()[pri:0, instance:com.xm.qa.test.AddGroupTest@4313f5bc]" name="addGroup" duration-ms="25769" started-at="2020-04-27T17:33:17Z" description="To verify that user is able to add the group" finished-at="2020-04-27T17:33:43Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.xm.qa.pages.AddGroupPage.enterGroupName(AddGroupPage.java:44)
	at com.xm.qa.test.AddGroupTest.addGroup(AddGroupTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addGroup -->
      </class> <!-- com.xm.qa.test.AddGroupTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
